sync/cli/awsemr.py

```diff
@@ -117,6 +118,12 @@ def create_submission(run_id: str, project: dict, region: str = None):
     return
 
 
+@aws_emr.command
+@click.argument("project", callback=validate_project)
+def create_recommendation(project: dict):
+    return create_project_recommendation(project["id"])
```

Comment:
Can we create a project recommendation for AWS EMR?

Reply:
Good point, we cannot at the moment. I can delete this. I think there are changes to endpoints that would need to happen to support this for aws_emr (I think it just has to switch the task name that gets invoked for aws_emr) @gorskysd thoughts?

Reply:
I think this has been taken care of already, but yes there's backend work that needs to be done to support emr.

Reply:
Deleted for now